src/App.svelte

```diff
@@ -1,7 +1,12 @@
 <script>
   import { exampleData } from "./ExampleData.js";
   export let servers = exampleData;
-  console.log(exampleData);
+  export let grabDomain = function(text){
+    const addresses = document.getElementsByClassName('address');
+    const toCopy = Array.from(addresses).find(({value}) => value === text);
```

Comment:
Instead of grabbing all of the address fields you should be binding the element. The copy element could even be its own component.

https://svelte.dev/tutorial/bind-this